poster/styles/styles.scss

```
@@ -0,0 +1,194 @@
+
+
+/*defining basics*/  
+
+* {background-color: #bc6c32;
+    font-family: sans-serif;
+    font-size: 32px;
+    font-weight: 200;}
+
+.wrapper {
+    background-color: #bc6c32;
+    display: -ms-grid;
+    display: grid;
+    -ms-grid-columns: 1fr 2fr 2fr;
+    grid-template-columns: 1fr 2fr 2fr;
+    grid-template-rows: .5fr 1.5fr 4fr;
+    margin: 5%;}
+
+/*order of styling:
+top
+side
+hero
+image*/
+
+/*organizing and styling the text at the top*/
+.top {
+    display: -ms-grid;
+    display: grid;
+   
+    grid-column: 1/4;
+    
+    grid-row: 1/2;
+   
+grid-template-columns: 1fr .32fr .32fr;}
+
+ h2 {
+font-size: 2.5vw;}
+
+p {font-size: 50%;
+    grid-column:2/4;
+    text-align: left;
+}
+
+/*organizing and styling the text to the side*/
+
+.side {
+grid-column: 1/2;
+grid-row:2/4;
+grid-template-rows: 1fr 1fr 1fr 1fr 1fr;
+margin-right: 20%;
+margin-top:0;}
+
+/*h3 span and h3 tageting different things*/
+h3 span {font-size: 40%;}
+
+h3 {margin-right:10%;}
+
+/*creating and placing the hero */
+.hero {grid-column: 2/4;
+}
+
+/*h1 span, nth child setup, and h1 tageting different things*/
+h1 {
+    display: grid;
+    grid-template-columns: .5fr .5fr;
+    padding:0;
+   text-transform: uppercase;}
+
+h1 :nth-child(2n) {
+  text-align: left;}
+
+h1 span {font-size:18vw;
+line-height: 100%;
+margin-top:0;
+}
+
+
+/*placing and sizing image*/
+.image {
+   align-content: right;
+    grid-column: 3/4;
+    -ms-grid-row: 2;
+    -ms-grid-row-span: 1;
+    grid-row: 2/3;
+    margin-top:7%;}
+
+img {
+    margin-left:2%;
+    max-width:100%;
+    overflow: hidden;
+    }
+   
+
+
+
+
+/* media queries*/
+
+
+@media (max-width:800px) and (min-width:500px) {
+
+    .wrapper {
+        grid-template-columns: 1fr 1.5fr;
+
+
+    }
+
+    .top {
+        display: grid;
+        grid-column: 1/3;
+        grid-template-columns: .5fr .5fr .5fr;
+        
+    }
+
+    h2 {font-size:75%;
+        margin-left: 10%; 
+    }
+
+    p {font-size: 1.75vw;
+        margin-left:5%;
+    }
+
+    .side {
+       
+        grid-column: 1/2;
+        grid-template-rows: .05fr .05fr .05fr .05fr .05fr;
+        margin-right: 20%;}
+
+    h3 span {display: block;font-size: 2.5vw;
+        font-size:100%;
+        line-height: 100%;}
+         margin-left:8%;
+        padding: 5%;
+       transform: rotate(0deg);}
+   
+   
+    .image {
+        grid-column: 2/3;
+        grid-row: 2/3;
+        object-fit: contain;}
+
+    
+    
+
+
+@media (max-width: 499px) and (min-width: 0px) {
```

You can leave off the last bit of this media query; the minimum would be implied 👍 

```suggestion
@media (max-width: 499px) {
```